Client/en.lproj/InfoPlist.strings

```diff
@@ -1,4 +1,4 @@
-"NSCameraUsageDescription" = "This lets you take and upload photos.";
+"NSCameraUsageDescription" = "To take and upload a photo to a website.";
```

Comment:
this change won't have any effect as it will get overridden by `./ios-l10n-scripts/import-locales-firefox.sh --release`

Reply:
Good call. We might need to have a script to change the imported ones if this keeps getting rejected.

Reply:
another option could be to not run this script anymore but have all the localised files in version control instead. That way changes in git will persist.